### PR TITLE
scientific use case section

### PR DESCRIPTION
**Pull request by elboyran**
Updating key-poitns of dianna

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,13 +68,14 @@ After studying the vast XAI landscape we have made choices in the parts of the [
...
 
-* Provides an easy-to-use interface for non (X)AI experts
-* Implements well-known XAI methods (LIME, RISE and Kernal SHAP) chosen by systematic and objective evaluation criteria
-* Supports the de-facto standard format for neural network models - ONNX.
-* Includes clear instructions for export/conversions from Tensorflow, Pytorch, Keras and scikit-learn to ONNX.
-* Supports images, text and time series data modalities. Tabular data and even embeddings support is planned.
-* Comes with simple intuitive image and text benchmarks
-* Easily extendable to other XAI methods
+ *   Provides an easy-to-use interface for non (X)AI experts
+ *   Implements well-known XAI methods (LIME, RISE and Kernal SHAP) chosen by systematic and objective evaluation criteria
```
Comment (SarahAlidoost) [inline README.md:72]:
```suggestion
 *   Implements well-known XAI methods LIME, RISE and Kernal SHAP, chosen by systematic and objective evaluation criteria
```

Comment (SarahAlidoost) [reply README.md:72]: Can we add a link to a reference that shows the "systematic and objective evaluation criteria"?

File: README.md [hunk 2]
```diff
@@ -68,13 +68,14 @@ After studying the vast XAI landscape we have made choices in the parts of the [
...
+ *   Supports the de-facto standard of neural network models - ONNX
+ *   Supports images, text, time series, and tabular data modalities, embeddings are currently being developed
+ *   Comes with simple intuitive image, text, time series, and tabular benchmarks, so can help you with your XAI research
+ *   Scientific use-cases tutorials
```
Comment (SarahAlidoost) [inline README.md:76]:
```suggestion
 *   Includes scientific use-cases tutorials
```

File: README.md [hunk 3]
```diff
@@ -320,10 +321,23 @@ DIANNA supports different data modalities and XAI methods (explainers). We have
...
+
+
+### Scientific use-cases
+Our goal is that the scientific community embrases XAI as a source for novel and unexplored perspectives on scientific problems. 
+Here, we offer [tutorials](./tutorials) on specific scientific use-cases of uisng XAI:
+| Use-case (data) \ XAI | [RISE](http://bmvc2018.org/contents/papers/1064.pdf) | [LIME](https://www.kdd.org/kdd2016/papers/files/rfp0573-ribeiroA.pdf) | [KernelSHAP](https://proceedings.neurips.cc/paper/2017/file/8a20a8621978632d76c43dfd28b67767-Paper.pdf) |
+| :---------                                           | :-------- | :------------------------------ | :-------------------------- |
+| Biology (Phytomorphology): Tree Leaves classification (images)   |        |            ✅                     |                             |
+| Astronomy: Fast Radio Burst detection (timeseries)    | ✅       |                                 |                             |
+| Geo-science (images)       |   planned                |  ...      | ...                            |                  ...         |
```
Comment (SarahAlidoost) [inline README.md:338]:
```suggestion
| Geo-science (raster data)       |   planned                |  ...      | ...                            |                  ...         |
```

File: tutorials/README.md [hunk 4]
```diff
@@ -1,71 +1,103 @@
...
 | Hyperparameter  | Default value |  <img width="20" alt="LeafSnap30 Logo" src="https://user-images.githubusercontent.com/3244249/151539100-dbdfe0f8-485f-45d4-a249-a1f79e970066.png"> (*i*) |<img width="25" alt="Weather Logo" src="https://github.com/dianna-ai/dianna/assets/3244249/3ff3d639-ed2f-4a38-b7ac-957c984bce9f"> (*ts*)| <img width="25" alt="Coffe Logo" src="https://github.com/dianna-ai/dianna/assets/3244249/9ab50a0f-5da3-41d2-80e9-70d2c8769162">(*ts*)|
 | ------------- | ------------- |--------| -----| -----|
 | $n_{samples}$  | **$5000$**  | $1000$ | $10 000$| $500$|
-| Kernel Width | **$25$**| default | default| default|
+| *Kernel Width* | **$25$**| default | default| default|
 | $n_{features}$ | **$10$** | $30$ | default| default|
 
+🠊 The most crucial parameter is the *Kernel width*: low values cause high sensitivity, however that observaiton was dependant on the evaluaiton metric.
+
 #### KernalSHAP
```
Comment (SarahAlidoost) [inline tutorials/README.md:94]:
```suggestion
#### KernelSHAP
```

**Review by SarahAlidoost — APPROVED** [review]
@elboyran thank you. I left some minor comments and approved this :+1:

File: README.md (hunk 1 above)
Comment (elboyran) [reply README.md]: It is mentioned in section Tutorials. The link is to our blog-post.

**Pull request merged**